.docker/entrypoint/spec/gemfile_generator_spec.rb

```diff
@@ -4,15 +4,15 @@
 require 'bundler'
 require 'gemfile_generator'
 require 'file_not_found_error'
-require 'be_valid_gemfile_matcher'
+require 'matchers/be_valid_gemfile_matcher'
 
 describe Jekyll::PlantUml::GemfileGenerator do
   subject(:generator) { Jekyll::PlantUml::GemfileGenerator.new }
 
   describe '#generate' do
-    let(:user_gemfile_path) { File.join(__dir__, 'Gemfile.user') }
-    let(:default_gemfile_path) { File.join(__dir__, 'Gemfile.default') }
-    let(:generated_gemfile_path) { File.join(__dir__, 'Gemfile.generated') }
+    let(:user_gemfile_path) { File.join(__dir__, 'data', 'Gemfile.user') }
+    let(:default_gemfile_path) { File.join(__dir__, 'data', 'Gemfile.default') }
+    let(:generated_gemfile_path) { File.join(__dir__, 'data', 'Gemfile.generated') }
```

Comment:
Codacy found an issue: [Line is too long. [84/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5768825)